### ISSUE TITLE
Update to Lando 3.20.7 is breaking php

### ISSUE THREAD

**Issue by mmodler** [lando/wordpress#56]
With 3.20.7, the following lando nginx+php setup becomes unuseable:

WARN ==> php is not a supported service type.
ERROR ==> Could not find a service builder called php in the builder registry!

```
name: test
services:
  appserver:
    type: php:8.1
    via: nginx:1.18
    ssl: true
    composer_version: 2
    webroot: public
    xdebug: off
    scanner: false
    config:
      vhosts: .lando/nginx.conf
      php: .lando/php.ini
  appserver_nginx:
    scanner: false
  redis:
    type: redis
    portforward: 6379
tooling:
  composer:
    service: appserver
    cmd: php /usr/local/bin/composer
  php:
    service: appserver
  redis-cli:
    service: redis
proxy:
  appserver_nginx:
    - aaa.local
    - bbb.local

```

**Comment by mmodler** [comment]
Sorry for posting to the wrong issue tracker - came here searching the error message :) Feel free to move the issue.

**Comment by mmodler** [comment]
Fixed with 3.20.8 - Thanks!
